srcpkgs/factor/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'factor'
 pkgname=factor
-version=0.97
+version=0.98
 revision=1
 only_for_archs="i686 x86_64 x86_64-musl"
 build_style=gnu-makefile
 hostmakedepends="unzip pkg-config"
-makedepends="gtkglext-devel"
+depends="gtk+-devel gtkglext-devel"
```

Comment:
what ? why are you adding development packages to runtime depdencnies ?

Reply:
@maxice8 They are required for the included GUI interface to the vm.